example/src/example_default.erl

```diff
@@ -0,0 +1,39 @@
+-module(example_default).
+
+-export(
+   [
+    init/3,
+    rest_init/2,
+    content_types_accepted/2,
+    content_types_provided/2,
+    forbidden/2,
+    resource_exists/2
+   ]
+  ).
+
+%% cowboy
+init(_Transport, _Req, _Opts) ->
+  {upgrade, protocol, cowboy_rest}.
+
+rest_init(Req, _Opts) ->
+  {ok, Req, #{}}.
+
+content_types_accepted(Req, State) ->
+  {Method, Req1} = cowboy_req:method(Req),
+  HandleMethod = case Method of
+                     <<"PUT">> ->
+                         handle_put;
+                     <<"POST">> ->
+                         handle_post
+                 end,
+  {[{<<"text/plain">>, HandleMethod}], Req1, State}.
+
+content_types_provided(Req, State) ->
+  {[{{<<"application">>, <<"text/plain">>, []}, handle_get}], Req, State}.
```

Comment:
should be like the one in `content_types_accepted`: `text/plain`, not `application/text/plain`